tsconfig.json

```diff
@@ -15,7 +15,8 @@
       "es2016",
       "dom",
       "es2017.object",
-      "es2016.array.include"
+      "es2016.array.include",
+      "ESNext.String"
```

Comment:
It may be a good idea to add a polifill for this to make sure it works on older browsers.

Reply:
I switched to replace() to avoid this change to tsconfig.